Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -2586,9 +2586,17 @@ def add_comment_command(client: JiraBaseClient, args: Dict[str, str]) -> Command
     visibility = args.get(
         "visibility",
     )
+    internal = argToBoolean(args.get("internal_comment"))
```

Comment:
```suggestion
    internal = argToBoolean(args.get("internal_comment", "false"))
```
to keep the original functionality valid